src/homeblks/volume/tests/vol_gtest.cpp

```diff
@@ -2223,6 +2447,41 @@ TEST_F(VolTest, init_io_test) {
     this->shutdown();
     if (tcfg.remove_file_on_shutdown) { this->remove_files(); }
 }
+TEST_F(VolTest, thin_test) {
```

Comment:
Can you put some comment how is test case trying to do? I am not able to understand the flow of this test case, e.g. what is the tuple meaning of each field, probably with proper comment around these fields, I can be able to understand better.